### PR TITLE
migrate docs to use firestore

### PR DESCRIPTION
**Pull request by Hyperkid123**
I've created API endpoint for the notifications in firebase functions
No more ambiguous responses we will always receive an array

Also the expiration is simplified. No more start date. Just expiration date. The panel will show everything that has expiration date larger than the current browser date.

### CODE REVIEW

File: packages/react-renderer-demo/src/components/notification-panel.js [hunk 1]
```diff
@@ -37,38 +37,27 @@ const useStyles = makeStyles((theme) => ({
...
-        return [];
-      }
-
-      if (!Array.isArray(data)) {
-        data = Object.values(data);
-      }
-
-      return data.filter(Boolean).sort((a, b) => b['created-at'].localeCompare(a['created-at']));
-    });
+    .then((data) => data);
```
Comment (rvsia) [inline packages/react-renderer-demo/src/components/notification-panel.js:42]: ^

File: packages/react-renderer-demo/firebaseFunctions.js [hunk 2]
```diff
@@ -13,12 +14,43 @@ const nextjsServer = next({
 });
 const nextjsHandle = nextjsServer.getRequestHandler();
 
+/**
+ * Initialize access to firestore
```
Comment (rvsia) [inline packages/react-renderer-demo/firebaseFunctions.js:18]: nitpick: I don't think this comment is useful :D

File: packages/react-renderer-demo/src/components/notification-panel.js [hunk 3]
```diff
@@ -37,38 +37,27 @@ const useStyles = makeStyles((theme) => ({
...
+        setNewMessages(data.filter((notification) => !lastSeen.includes(createNotificationId(notification))).length);
+        localStorage.setItem('data-driven-forms-last-seen', JSON.stringify(data.map(createNotificationId)));
+        setNotifications(data);
+      })
+      .catch(() => {
+        setNotifications([]);
+      });
   }, [setNewMessages]);
 
   return (
```
Comment (rvsia) [inline packages/react-renderer-demo/src/components/notification-panel.js:63]: I am sorry, there is actually `created-at` in the old UI code, not `created_at` :-/ So you can change it here.

Comment (Hyperkid123) [reply packages/react-renderer-demo/src/components/notification-panel.js:59]: done

**Pull request merged**